docroot/profiles/custom/cgov_site/themes/custom/ncids_trans/templates/listing/list_item_macros.twig

```diff
@@ -225,15 +225,15 @@
     <span class="media-type">({{ media_link_type|trans }})</span>
   {% endif %}
   {% if media_link_type == "File" %}
-    <span class="filetype {{file_type}}">
+    <span class="filetype">[{{file_type}}
       {% if file_type != "unknown" %}
         {# Note, the spanish should have the word order reversed... #}
         <span class="show-for-sr">{{ file_type }} {% trans %}file{% endtrans %}</span>
       {% else %}
         {{ file_ext }}<span class="show-for-sr"> {% trans %}file{% endtrans %}</span>
       {% endif %}
```

Comment:
It does not seem like this conditional is needed anymore. In the DOM you would be outputting the text twice.